modules/processors/frame/core.py

```diff
@@ -40,22 +40,29 @@ def get_frame_processors_modules(frame_processors: List[str]) -> List[ModuleType
     set_frame_processors_modules_from_ui(frame_processors)
     return FRAME_PROCESSORS_MODULES
 
+
 def set_frame_processors_modules_from_ui(frame_processors: List[str]) -> None:
     global FRAME_PROCESSORS_MODULES
     for frame_processor, state in modules.globals.fp_ui.items():
-        if state == True and frame_processor not in frame_processors:
+        if state and frame_processor not in frame_processors:
             frame_processor_module = load_frame_processor_module(frame_processor)
             FRAME_PROCESSORS_MODULES.append(frame_processor_module)
             modules.globals.frame_processors.append(frame_processor)
-        if state == False:
+        if not state:
             try:
                 frame_processor_module = load_frame_processor_module(frame_processor)
                 FRAME_PROCESSORS_MODULES.remove(frame_processor_module)
                 modules.globals.frame_processors.remove(frame_processor)
-            except:
+            except:  # noqa: B001
```

Comment:
I tried to catch this particulair exception, but when I do, the application won't start. I've made an ignore case for it by adding `noqa: B001`.

Reply:
I would rather skip this for now, you already complained about the fact that this PR is big, and this goes deeper because this exception raises another one somewhere in the code. It was never my intention to fix this to begin with.